include/jemalloc/internal/mutex_inlines.h

```diff
@@ -22,12 +25,38 @@ malloc_mutex_lock_final(malloc_mutex_t *mutex) {
 	MALLOC_MUTEX_LOCK(mutex);
 }
 
-/* Trylock: return false if the lock is successfully acquired. */
 JEMALLOC_INLINE bool
-malloc_mutex_trylock(malloc_mutex_t *mutex) {
+malloc_mutex_trylock_final(malloc_mutex_t *mutex) {
 	return MALLOC_MUTEX_TRYLOCK(mutex);
 }
 
+JEMALLOC_INLINE void
+mutex_owner_stats_update(tsdn_t *tsdn, malloc_mutex_t *mutex) {
+	if (config_stats) {
+		mutex_prof_data_t *data = &mutex->prof_data;
+		data->n_lock_ops++;
+		if (data->prev_owner != tsdn) {
+			data->prev_owner = tsdn;
+			data->n_owner_switches++;
+		}
+	}
+}
+
+/* Trylock: return false if the lock is successfully acquired. */
+JEMALLOC_INLINE bool
+malloc_mutex_trylock(tsdn_t *tsdn, malloc_mutex_t *mutex) {
+	bool ret = false;
```

Comment:
Style nit: move initialization to an ```else``` block to avoid double initialization, or do early ```return false``` and refactor ```ret``` out.